apps/site/assets/css/_trip-plan-widget.scss

```diff
@@ -104,9 +116,138 @@
   justify-content: flex-start;
 }
 
-.c-trip-plan-widget__reverse-control {
+.c-trip-plan-widget__reverse-control-vert {
   color: $brand-primary;
   cursor: pointer;
   margin-left: auto;
   user-select: none;
 }
+
+.col-xs-12 {
```

Comment:
![image](https://user-images.githubusercontent.com/526017/170775854-be637455-74a2-4fe9-a616-36a8477a2f54.png)

if you go to /trip-planner the arrows aren't being hidden at the different resolutions.